karma.conf.js

```diff
@@ -14,6 +14,15 @@ module.exports = function (config) {
       'expect',
       'mocha'
     ],
+    plugins: [
+      'karma-browserify',
+      'karma-chrome-launcher',
+      'karma-phantomjs-launcher',
+      'karma-expect',
+      'karma-mocha',
+      'karma-spec-reporter',
+      require('@coderbyheart/karma-sauce-launcher')
```

Comment:
The plugin is named 'karma-sauce-launcher', but the package is named something else, so we have to do this.  Unfortunately, if you start specifying plugins, you have to specify *all* of them.